src/index.js

```diff
@@ -555,9 +556,20 @@ const Instauto = async (db, browser, options) => {
       if (!foundClickable) throw new Error('Like button not found');
 
       if (!dryRunIn) {
-        foundClickable.click();
+        const img = dialog.querySelector('img[srcset]');
```

Comment:
I think it's not very safe to just match **any** image. as it could also match other images on the page. why are you matching `srcset`?

I think a safer way to match could be a hierarchy this:
```html
<div role="dialog">
<article role="presentation">
<img alt> where starts with "Photo by "
```